### PR TITLE
Add October 2020 Recap Blog Post

### PR DESCRIPTION
**Pull request by seand7565**

### CODE REVIEW

File: source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md
Comment (aldesantis) [inline source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md:30]: I think here you meant "country's" instead of "countries" and "Italy's" instead of "Italys."

File: source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md
Comment (aldesantis) [inline source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md:32]: "would fail" is missing a subject I think. Shouldn't it be "the process would fail" or similar?

**Review by aldesantis — APPROVED** [review]
@seand7565 just a couple of things to fix, but I'm approving so that you can merge right after fixing them. 🙂

File: source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md
Comment (seand7565) [reply source/blog/2020-11-10-this-month-in-solidus-october-2020.html.erb.md:32]: Good catch, thanks!

**Pull request merged**
